sdk/cognitiveservices/ms-azure-cs-inkrecognizer/inkrecognizer/src/main/java/com/azure/ai/inkrecognizer/InkRecognizerClient.java

```diff
@@ -0,0 +1,196 @@
+     * needed when the language is different from the default set when the client was instantiated.
+     * @return An InkRecognitionResult containing status codes, error messages (if applicable) and the recognition results
+     * in a hierarchy.
+     * @throws Exception
+     */
+    public Response<InkRecognitionRoot> recognizeInk(
+            Iterable<InkStroke> strokes,
+            String language
+    ) throws Exception {
+        return recognizeInk(
+                strokes,
+                this.unit,
+                this.unitMultiple,
+                this.applicationKind,
+                language,
+                this.displayMetrics
+        );
+    }
+
+    /**
+     * Synchronously sends data to the service and generates a tree structure containing the recognition results.
+     * @param strokes The ink strokes to recognize.
+     * @param unit The physical unit for the points in the stroke.
+     * @param multiple A multiplier applied to the unit value to indicate the true unit being used. This allows the
+     * caller to specify values in a fraction or multiple of a unit.
+     * @param applicationKind The domain of the application (Writing or Drawing. The default is "Mixed").
+     * @param language The IETF BCP 47 language code (for ex. en-US, en-GB, hi-IN etc.) for the strokes. This is only
+     * needed when the language is different from the default set when the client was instantiated.
+     * @param displayMetrics A structure describing general information about the display, such as xdpi, ydpi, where the
+     * application using this SDK will run on.
+     * @return A InkRecognitionResult containing status codes, error messages (if applicable) and the recognition results
+     * in a hierarchy.
+     * @throws Exception
+     */
+    public Response<InkRecognitionRoot> recognizeInk(
+            Iterable<InkStroke> strokes,
+            InkPointUnit unit,
+            float multiple,
+            ApplicationKind applicationKind,
+            String language,
+            DisplayMetrics displayMetrics
+    ) throws Exception {
+
+        String requestJSON = Utils.createJSONForRequest(strokes, unit, multiple, applicationKind, language, displayMetrics);
+
+        MediaType mediaTypeJSON = MediaType.parse("application/json; charset=utf-8");
+        OkHttpClient client = httpClient.newBuilder()
+                .connectTimeout(retryTimeout, TimeUnit.MILLISECONDS)
+                .build();
+        RequestBody body = RequestBody.create(requestJSON, mediaTypeJSON);
+        Request request = new Request.Builder()
+                .url(endpoint + serviceVersion.toString())
+                .put(body)
+                .build();
+        request = credentials.SetRequestCredentials(request);
+
+        for (int retryAttempt = 0; retryAttempt < retryCount; ++retryAttempt) {
+            try (okhttp3.Response response = client.newCall(request).execute()) {
+                if (response.code() != 200) {
```

Comment:
These 8 lines of code could be just a if (... == 200) here, and after the for loop, a single line with throw new Exception.

Reply:
Won't the information in 'response' be lost then?

Reply:
Fixed.